pyeudiw/storage/db_engine.py

```diff
@@ -39,15 +42,24 @@ def _handle_instance(instance: dict) -> dict[BaseStorage | None, BaseCache | Non
     
     def init_session(self, dpop_proof: dict, attestation: dict):
         document_id = str(uuid.uuid4())
-        for storage in self.storages:
-            storage.init_session(dpop_proof, attestation)
+        for db_name, storage in self.storages:
+            try:
+                storage.init_session(dpop_proof, attestation)
+            except Exception as e:
+                logger.critical("Cannot write document with id {document_id} on {db_name}")
             
         return document_id
 
-    def update_request_object(self, document_id: str, request_object: dict):
-        for storage in self.storages:
-            storage.update_request_object(document_id, request_object)
+    def update_request_object(self, document_id: str, nonce: str, state: str, request_object: dict):
```

Comment:
state is optional

Reply:
Resolved in commit [9386806](https://github.com/italia/eudi-wallet-it-python/pull/67/commits/9386806d0f517ab0d729b7ac0d79e095f757ff53)